pyro/distributions/__init__.py

```diff
@@ -33,6 +34,7 @@
     "Rejector",
     "TorchDistribution",
     "VonMises",
+    "RelaxedOneHotCategoricalStraightThrough"
```

Comment:
nit: make these alphabetical and add a trailing comma.

Reply:
done